unity/Assets/Scripts/Unity/UnityStack.cs

```diff
@@ -56,7 +56,7 @@ private void DisplayStack()
             {
                 ITile tile = firstTen.ElementAt(i);
                 Texture texture = GetTextureForTileType(tile);
-                GameObject newTile = Instantiate(Template, Content.transform);
+                GameObject newTile = Instantiate(Template);
```

Comment:
What does this change? Just curious.

Reply:
I had a warning telling me there are too many transforms going on, removing this parameter fixed it.